modules/reports/server-ts/pdf/helpers/PDFBuilder.ts

```diff
@@ -0,0 +1,56 @@
+import PdfPrinter from 'pdfmake';
+import fonts from './fonts/Roboto/';
+
+export default class PDFBuilder {
+  private printer: any;
```

Comment:
Types

Reply:
Partially Fixed in #1023, printer is third-party library, so we can't add types for it.

Reply:
@Sasha-belokur Why? How about:
https://www.npmjs.com/package/@types/pdfmake

Reply:
I added types from package above. But I couldn't add type for server printer object since these types are for front-end library. We discussed with @lyzhovnik, that it would be better to get rid of constructor and assign properties on declaration phase.